salt/modules/status.py

```diff
@@ -1053,11 +1053,14 @@ def diskusage(*args):
     ret = {}
     for path in selected:
         if os.path.exists(path):
-            fsstats = os.statvfs(path)
-            blksz = fsstats.f_bsize
-            available = fsstats.f_bavail * blksz
-            total = fsstats.f_blocks * blksz
-            ret[path] = {"available": available, "total": total}
+            try:
+                fsstats = os.statvfs(path)
+                blksz = fsstats.f_bsize
+                available = fsstats.f_bavail * blksz
+                total = fsstats.f_blocks * blksz
+                ret[path] = {"available": available, "total": total}
+            except OSError as exc:
+                log.warning("Cannot get stats from '{}': {}".format(path, exc))
```

Comment:
So personally, I would expect something like:

```suggestion
                log.warning("Cannot get stats from '{}': {}".format(path, exc))
                ret[path] = {"available": None, "total": None}
```
Or something similar to let the user know there was an error right away with the path. But, not sure, if upstream is OK with it, why not.

Reply:
Thanks for the comment. Indeed, it sounds reasonable, and I also like it more than the current approach.

I'll include your suggestion here and also to the upstream PR :+1: